flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.ts

```diff
@@ -61,7 +67,22 @@ export class JobExceptionsComponent implements OnInit {
           this.rootException = 'No Root Exception';
         }
         this.truncated = exceptionHistory.truncated;
-        this.listOfException = exceptionHistory.entries;
+        this.listOfException = exceptionHistory.entries.map(entry => {
+          const values: ExceptionInfoInterface[] = [];
+          values.push(entry);
```

Comment:
I think this will add all concurrent exceptions to this entry as well, which will be unnecessary clutter (have yet to find out how I can produce nice exceptions to test...)

We can use something like this to extract the main exception:
```
function extractMainException(rootException: RootExceptionInfoInterface) {
    const {concurrentExceptions, ...mainException} = rootException;
    return mainException;
}
```
I first wanted to use lodash fp `omit`, but it only gives `Partial` return types, which is annoying